### ISSUE TITLE
Mark Record "provisional" when created.. View should only show published records.

### ISSUE THREAD

**Issue by thoufer** [thoufer/mbp-metadatacatalog#65]

**Comment by thoufer** [comment]
@embuco .. Been thinking about this issue some.  I can envision the problem where we have simply 2 states; new/provisional and complete/published.  Should that be all?  Do we ned to consider the a case an asset is "in review", but that review hasn't been completed.  What about a case where we are unable to finalize an asset because information is missing, incorrect, or incomplete?

**Comment by thoufer** [comment]
If a record is modified it should probably be changed back to provisional, and an email sent that includes the the original data and the modified data?

**Comment by embuco** [comment]
@thoufer good point, although I don't think we want to create too many categories ... how about 3? "complete" "incomplete" and "provisional" .... the difference being that provisional = still need to decide if it should be in the catalog and incomplete is "yes, but needs some sort of work" while "complete" is just that! .... the only issue is the set that we might leave in but not consider to be MBP assets. I guess that is a separate field Yes/No as MBP data asset?

**Comment by thoufer** [comment]
@embuco... I think there is some confounding going on here.  We have a column that the user sets as to whether the the FWS is the/an authoritative source, which determines if the record in an MBP Asset.  Then we have a second column that manages the record state ...[new/created, Reviewed, Complete, incomplete].  It seems that you want a workflow that also includes some sort of decision on our part as to whether the record should be considered an MBP asset (e.g., different terminal points complete asset, complete not asset).

**Comment by embuco** [comment]
@thoufer , yes, well I suppose the user yes/no is to try to curate the asset as it is entered (avoid stuff we don't want by making the person affirm that it goes in the catalog). The "provisional" is b/c we want a general review of appropriateness (including the answer to that field) by some authority. That authority can change the field to NO, if needed.
I think the 2nd column has 3: new/created, incomplete, and complete .... if it is "reviewed" it would be moved into either complete or incomplete and the MBP asset YES/NO would be reviewed as part of that process.
